website/public/index.ejs.html

```diff
@@ -802,6 +802,128 @@ <h2 id="linter-feature-comparison">Features</h2>
+                >
+              </td>
+              <td class="meh">
+                <span aria-labelledby="feature-speed-eslint-capable">🐇</span
+                ><br /><span id="feature-speed-eslint-capable">capable</span>
+              </td>
+              <td class="bad">
+                <span aria-labelledby="feature-speed-jshint-slow">🐢</span
+                ><br /><span id="feature-speed-jshint-slow">slow</span>
+              </td>
+            </tr>
+            <tr>
+              <td class="good quick-lint-js">
+                <span aria-label="easy">✅</span><br />zero config
+              </td>
+              <td class="meh">
+                <span aria-label="hard">❌</span><br />config or<br />
+                <code>init</code>
+              </td>
+              <td class="bad"><span aria-label="hard">❌</span><br />config</td>
+            </tr>
+
+            <tr>
+              <td class="good quick-lint-js">free<br />GPLv3</td>
+              <td class="meh">
+                open<br />
+                source<br />MIT
+              </td>
+              <td class="meh">
+                open<br />
+                source<br />MIT
+              </td>
+            </tr>
+            <tr>
+              <td class="good quick-lint-js">
+                <span aria-label="yes">✅<br />JSX</span>
+              </td>
+              <td class="meh">
+                <span aria-label="yes">✅<br />JSX</span>
+              </td>
+              <td class="bad">
+                <span aria-label="no">❌<br />JSX</span>
+              </td>
+            </tr>
+            <tr>
+              <td class="bad quick-lint-js">
+                <span aria-label="no">❌<br />TypeScript</span>
+              </td>
+              <td class="meh">
+                <span aria-label="yes">✅<br />TypeScript</span>
+              </td>
+              <td class="bad">
+                <span aria-label="no">❌<br />TypeScript</span>
+              </td>
+            </tr>
+
+            <tr>
+              <td class="meh quick-lint-js">
+                <a href="errors/">minimal</a><br />Lint Rules
+              </td>
+              <td class="good">
+                <a href="https://eslint.org/docs/rules/">zillions</a><br />Lint
+                Rules
+              </td>
+              <td class="meh">
+                <a href="https://jshint.com/docs/options/">some</a><br />Lint
+                Rules
+              </td>
+            </tr>
+            <tr>
+              <td class="good quick-lint-js">
+                <span aria-label="stable">✅</span><br />Stable
+              </td>
+              <td class="good">
+                <span aria-label="stable">✅</span><br />Stable
+              </td>
+              <td class="good">
+                <span aria-label="stable">✅</span><br />Stable
+              </td>
+            </tr>
+          </tbody>
+        </table>
```

Comment:
This information seems to be duplicated. For example, when we implement TypeScript support in quick-lint-js, I need to remember to update two tables. (I'll probably forget to update the mobile version!)

Can we reuse the existing markup? Alternatively, could we generate both tables from a single source of data? (We can use EJS here.)